service/controller/resource/alerting/heartbeatwebhookconfig/resource.go

```diff
@@ -54,7 +54,7 @@ func New(config Config) (*generic.Resource, error) {
 	return r, nil
 }
 
-func getObjectMeta(v interface{}, config Config) (metav1.ObjectMeta, error) {
+func getObjectMeta(v interface{}) (metav1.ObjectMeta, error) {
```

Comment:
What's the point of this ?

Reply:
It gave me an unused warning

Reply:
I'm doing it in the follow up PR anyway :D